shell/platform/android/test/io/flutter/embedding/android/FlutterViewTest.java

```diff
@@ -375,7 +375,11 @@ public void systemInsetHandlesFullscreenNavbarLeft() {
   public void flutterImageView_acquiresImageAndInvalidates() {
     final ImageReader mockReader = mock(ImageReader.class);
     final FlutterImageView imageView =
-        spy(new FlutterImageView(RuntimeEnvironment.application, mockReader));
+        spy(
```

Comment:
`FlutterViewTest.flutterImageView_acquiresImageAndInvalidates` is failing on LUCI

Reply:
Fixed.